fgrpc/ping_grpc.pb.go

```diff
@@ -0,0 +1,101 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```

Comment:
shouldn't be a new file... there is already  ping.pb.go

Reply:
The protoc commands as originally suggested in the proto file don't work with the latest protoc (3.17.3)

And newer versions split the gRPC parts into their own file.